packages/interface-stream-muxer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@libp2p/interface-stream-muxer",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
Should we avoid this too for the same reason as: https://github.com/libp2p/js-libp2p-interfaces/pull/290#discussion_r987048662 ?

Reply:
ah, sorry. Will fix